[2025-02-14 03:15:48,314] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: Project.task_three 2025-02-14T03:11:16.276498+00:00 [queued]>
[2025-02-14 03:15:48,325] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: Project.task_three 2025-02-14T03:11:16.276498+00:00 [queued]>
[2025-02-14 03:15:48,326] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2025-02-14 03:15:48,326] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2025-02-14 03:15:48,326] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2025-02-14 03:15:48,342] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): task_three> on 2025-02-14T03:11:16.276498+00:00
[2025-02-14 03:15:48,347] {standard_task_runner.py:52} INFO - Started process 1626 to run task
[2025-02-14 03:15:48,352] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Project', 'task_three', '2025-02-14T03:11:16.276498+00:00', '--job-id', '131', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpoeqr3chd', '--error-file', '/tmp/tmp4ghnbcjl']
[2025-02-14 03:15:48,353] {standard_task_runner.py:77} INFO - Job 131: Subtask task_three
[2025-02-14 03:15:48,403] {logging_mixin.py:104} INFO - Running <TaskInstance: Project.task_three 2025-02-14T03:11:16.276498+00:00 [running]> on host 9e3f73609008
[2025-02-14 03:15:48,457] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Project
AIRFLOW_CTX_TASK_ID=task_three
AIRFLOW_CTX_EXECUTION_DATE=2025-02-14T03:11:16.276498+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-14T03:11:16.276498+00:00
[2025-02-14 03:15:58,252] {logging_mixin.py:104} INFO - wrote file: titles.json
[2025-02-14 03:15:58,252] {python.py:151} INFO - Done. Returned value was: None
[2025-02-14 03:15:58,265] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=Project, task_id=task_three, execution_date=20250214T031116, start_date=20250214T031548, end_date=20250214T031558
[2025-02-14 03:15:58,293] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-14 03:15:58,344] {local_task_job.py:151} INFO - Task exited with return code 0
